test/partiql-randomized-tests/src/test/kotlin/org/partiql/lang/randomized/eval/EvaluatingCompilerIntRandomizedTest.kt

```diff
@@ -16,17 +16,20 @@
 package org.partiql.lang.randomized.eval
 
 import org.junit.jupiter.api.BeforeAll
-import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.MethodSource
 import java.util.Random
 
 /**
  * This class tests evaluation-time behavior for integer and integer overflows that existed *prior* to the
- * introduction of StaticType.  The behavior described in these tests is still how the we should handle
- * integer arithmetic in the absence of type information.
+ * introduction of StaticType. The behavior described in these tests is still how we should handle integer arithmetic
+ * in the absence of type information.
+ *
+ * TODO int overflow behavior is confusing. Should we
+ *  - Overflow (no error)
+ *  - Upcast to long (no error)
+ *  - Error
```

Comment:
(self-review) cut an issue to track -- https://github.com/partiql/partiql-lang-kotlin/issues/1697.

Will discuss w/ team on whether the evaluation behavior needs changed. If so, how to update the randomized tests.

Reply:
These tests and implementation should give an error. I've marked these tests as disabled for now until we fix #1697.